manager/src/main/java/io/aklivity/zilla/manager/internal/commands/install/ZpmInstall.java

```diff
@@ -548,7 +549,7 @@ private void generateDelegate(
                                 String serviceName = servicePath.toString();
                                 String serviceImpl = new String(input.readAllBytes(), UTF_8);
                                 String existing = services.getOrDefault(serviceName, "");
-                                services.put(serviceName, existing.concat(serviceImpl));
+                                services.put(serviceName, existing.concat("\n").concat(serviceImpl));
```

Comment:
Let's change this so it produces the same output regardless of whether or not a newline is present in the input.

For example, we can collect the implementations as a list of implementation class names (without whitespace) instead of a concatenated string here, and when writing out, we can join on newline via
```
String.join("\n", serviceImpls)
```